sftp-client.c

```diff
@@ -1140,6 +1155,20 @@ send_read_request(struct sftp_conn *conn, u_int id, u_int64_t offset,
 	sshbuf_free(msg);
 }
 
+#ifdef IRONCORE
+static char
+get_yes_no(void)
+{
+	char line[30];
+	fgets(line, sizeof(line), stdin);
+	char yesno = line[0];
+	while (line[strlen(line) - 1] != '\n') {
+		fgets(line, sizeof(line), stdin);
+	}
+	return yesno;
+}
```

Comment:
It seems like you ought to include the logic to check if `yesno` is `Y` or `y` in here and just return a boolean-ish. It sure looks like that's how you actually use it.